onset_maproom/config-nma-ethiopia.yaml

```diff
@@ -52,7 +52,7 @@ name_adm3: Woredas
 
 # Onset_and_Cessation
 app_title: Growing Season Maproom
-onset_and_cessation_title: Planting and Harvest Decision Support Maproom
+onset_and_cessation_title: Planting Date Decision Support Maproom
```

Comment:
Tufa wanted that change? It's not very indicative that there is cessation date shown in the App too...

Reply:
That is what he asked it to be named last I spoke with him.